pkg/controller/networkpolicy/group.go

```diff
@@ -244,3 +244,10 @@ func (n *NetworkPolicyController) updateGroupStatus(g *crdv1beta1.Group, cStatus
 	_, err := n.crdClient.CrdV1beta1().Groups(g.GetNamespace()).UpdateStatus(context.TODO(), toUpdate, metav1.UpdateOptions{})
 	return err
 }
+
+// GetGroupMembers returns the current members of a Group.
+// If the Group is defined with IPBlocks, the returned members will be []controlplane.IPBlock.
+// Otherwise, the returned members will be of type controlplane.GroupMemberSet.
+func (n *NetworkPolicyController) GetGroupMembers(name, namespace string) (controlplane.GroupMemberSet, []controlplane.IPBlock, error) {
+	return n.GetClusterGroupMembers(namespace + "/" + name)
```

Comment:
This looks very confusing. We could keep the original function named `GetGroupMembers` and name this new one `GetGroupMembersByNamespace`?

Reply:
Makes sense, done.